src/jit/importer.cpp

```diff
@@ -18951,6 +18952,11 @@ bool Compiler::IsTargetIntrinsic(CorInfoIntrinsics intrinsicId)
         case CORINFO_INTRINSIC_Abs:
             return true;
 
+        case CORINFO_INTRINSIC_Round:
+        case CORINFO_INTRINSIC_Ceiling:
+        case CORINFO_INTRINSIC_Floor:
+            return JitTls::GetCompiler()->compSupports(InstructionSet_SSE41);
```

Comment:
Is there a better way to test/check this?

Reply:
What's `JitTls::GetCompiler` for?!

Reply:
It seems to be the way other methods are getting an instance of the compiler. I expect that this is not correct, which is why I explicitly called it out 😄 

`Round`, `Ceiling`, and `Floor` are only intrinsic if SSE4.1 is supported, so the check needs to be somewhere, just not sure where the best place is.

Reply:
But that code is in a `Compiler` member function, you already have an instance of the compiler.

Reply:
It's a static member.

Reply:
I might just be able to return `true` here and check in `impMathIntrinsic` instead, which is an instance method... Still investigating.

Reply:
Yeah, it seems like I can either:
* Return `true` in `IsTargetIntrinsic` and then have a sub-check in `impMathIntrinsic` for `compSupports`
-or-
* Change `IsTargetIntrinsic` (and the other three methods) to be instance methods

The latter (convert to instance methods) seems like the better option since the `compSupports` check won't have to be replicated everywhere required, and the `IsTargetIntrinsic` method will be self-consistent.

Reply:
I agree that it makes sense for these to be instance methods.